src/main/scala/chiseltest/simulator/VerilatorCirctSimulator.scala

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiseltest.simulator
+
+import chiseltest.internal.Utils.makeScriptFromCommand
+import firrtl._
+import firrtl.annotations._
+import chiseltest.simulator.jna._
+import firrtl.stage.OutputFileAnnotation
+
+case object VerilatorCirctBackendAnnotation extends SimulatorAnnotation {
+  override def getSimulator: Simulator = VerilatorCirctSimulator
+}
+
+private object VerilatorCirctSimulator extends VerilatorSimulatorTrait {
```

Comment:
I wonder if it would be easier to add a flag to the original `VerilatorSimulator` that switches the way that Verilog is generated.

Reply:
I actually started that way, but I think this is better to keep this more separate. I think I will probably be doing more work in this area. This keeps it clean and distinct

Reply:
So what is the plan for the other Verilog Simulators like icarus and VCS? Should the also have an option to use `firtool` instead of the firrtl compiler?